src/src/pages/capabilities/details.js

```diff
@@ -33,6 +33,7 @@ function CapabilityDetailsPageContent() {
     showResources,
     showCosts,
     isPendingDeletion,
+    isDeleted,
```

Comment:
At some point we should consider sending the state itself, instead of a host of boolean flags.

However, doing it this way allow for string comparison in just a single location which have some benefits.
Therefore I did not change it at this point in time.